### PR TITLE
Support to add annotation to member already annotated

### PR DESCRIPTION
**Pull request by fabapp2**

### CODE REVIEW

File: rewrite-test/src/main/kotlin/org/openrewrite/java/JavaTemplateTest.kt [hunk 1]
```diff
@@ -812,6 +812,73 @@ interface JavaTemplateTest : JavaRecipeTest {
...
+                    @SuppressWarnings("ALL") int m;
+                    final int n;
+                }
+            }
+        """,
+        after = """
+            class Test {
+                void test() {
+                    @SuppressWarnings("ALL")
+                    @Deprecated int m;
```
Comment (fabapp2) [inline rewrite-test/src/main/kotlin/org/openrewrite/java/JavaTemplateTest.kt:842]: I think here the formatting could be improved and add a linebreak after `@Deprecated`...
For our current requirement, the result is 'good enough' though. 
I can give it a try tomorrow if yo want me to?

Comment (jkschneider) [reply rewrite-test/src/main/kotlin/org/openrewrite/java/JavaTemplateTest.kt:842]: Yes, that would be nice. Ultimately, that blank line is the `suffix` on the last `Comment`

Comment (fabapp2) [reply rewrite-test/src/main/kotlin/org/openrewrite/java/JavaTemplateTest.kt:842]: I might not have understood, but I couldn't see how "blank line is the suffix on the last Comment" should work?

````
/** 
* some comment
*/
int a;
````
became 
````
/** 
* some comment
*/
@Deprecated int a;
````
and without an existing comment, I didn't get it?

However, I got most cases working and tested. I would have found a dedicated test class for formatting related Visitors  helpfull, but that's for another time ;)

Comment (fabapp2) [reply rewrite-test/src/main/kotlin/org/openrewrite/java/JavaTemplateTest.kt]: I would call it "done" (for now)? ;)

**Comment by tkvangorder** [comment]
Thanks for the contribution!

**Pull request merged**